src/pages/individual/Individual.jsx

```diff
@@ -12,6 +12,11 @@ import useIndividual from '../../models/individual/useIndividual';
 import useDeleteIndividual from '../../models/individual/useDeleteIndividual';
 import usePatchIndividual from '../../models/individual/usePatchIndividual';
 
+// VERILY BAD HOTFIX //
+import defaultIndividualSrc from '../../assets/defaultIndividual.png';
```

Comment:
???

Reply:
1. What's so bad about this?
2. Why is this a hotfix?

Reply:
hmm, really these should be "temporary fixes". i made some quick adjustments to render the individuals page (which currently is erroring out) so that i could get the bao working on that page. but i need to go back and do it the _right_ way once i move on to the individuals tickets 

these particular one is really bad because it's using components from the wrong page